generation_config.yaml

```diff
@@ -1,6 +1,6 @@
 gapic_generator_version: 2.37.0
 protobuf_version: '25.2'
-googleapis_commitish: 6500290663163ba7dc6e0a35231772f5f78c3b62
+googleapis_commitish: e960a82d36e3ddaeb62f549dbd4c300e11e240dc
```

Comment:
It shows as changing but it's the same as the latest:
https://github.com/googleapis/google-cloud-java/blob/14f7146c98fdf03de2f113215e03347a21e83f9a/generation_config.yaml#L3

Reply:
Can we revert this whole file?